webpack.config.js

```diff
@@ -1,135 +0,0 @@
-const path = require('path');
```

Comment:
Was this file unused?

Reply:
seems now we only use webpack.common.js webpack.prod.js and webpack.dev.js

Reply:
@jeremenichelli https://github.com/webpack/webpack.js.org/pull/2251

Reply:
👍